packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -81,33 +57,24 @@ const liquidate = async (
     }),
     vaultZcfSeat,
     vaultZcfSeat,
-    harden({ debt }),
+    harden({ debt, penaltyRate }),
   );
-  trace(` offeredTo`, collateralToSell, debt);
-
-  // await deposited, but we don't need the value.
-  await Promise.all([deposited, E(liqSeat).getOfferResult()]);
-
-  // Now we need to know how much was sold so we can pay off the debt.
-  // We can use this because only liquidation adds RUN to the vaultSeat.
-  const { debtPaid, penaltyProceeds, runToBurn } = partitionProceeds(
-    vaultZcfSeat.getAmountAllocated('RUN', debt.brand),
-    debt,
-    penalty,
-  );
-
-  trace({ debt, debtPaid, penaltyProceeds, runToBurn });
-
-  // Allocate penalty portion of proceeds to a seat that will be transferred to reserve
-  penaltyPoolSeat.incrementBy(
-    vaultZcfSeat.decrementBy(harden({ RUN: penaltyProceeds })),
-  );
-  zcf.reallocate(penaltyPoolSeat, vaultZcfSeat);
-
+  trace(` offeredTo`, { collateralToSell, debt });
+
+  // await deposited and offer result, but ignore the latter
+  const [proceeds] = await Promise.all([
+    deposited,
+    E(liqSeat).getOfferResult(),
+  ]);
+  // all the proceeds from AMM sale are on the vault seat instead of a staging seat
+  // ??? ^ okay?
```

Comment:
Yes, it's fine that all the proceeds end up on the VaultSeat. You can drop the `???` comment.